app/uk/gov/hmrc/gform/views/form/form.scala.html

```diff
@@ -124,25 +126,48 @@
       }
     }
 
-    @formWithoutCSRF(action = page.formAction, args = 'class -> "js-form", 'autocomplete -> "on", 'novalidate -> "novalidate", 'id -> "gf-form") {
+    @if(page.renderWithoutCSRF){
```

Comment:
This condition is redundant. It is same as `page.upscanUrl.isDefined`.
When we have  `page.upscanUrl` we want to render `"gf-upscan"`

Also `"gf-form"` form should be defined only once in this file.